From: Alberto Garcia <berto@igalia.com>
Subject: Set full path of the ocrfeeder icon
Forwarded: not-needed
Index: ocrfeeder/resources/ocrfeeder.desktop.in
===================================================================
--- ocrfeeder.orig/resources/ocrfeeder.desktop.in
+++ ocrfeeder/resources/ocrfeeder.desktop.in
@@ -5,6 +5,6 @@
 _Comment=The complete OCR suite.
 TryExec=ocrfeeder
 Exec=ocrfeeder %F
-Icon=ocrfeeder
+Icon=/usr/share/ocrfeeder/icons/ocrfeeder.svg
 MimeType=image/bmp;image/gif;image/jpeg;image/jpg;image/pjpeg;image/png;image/tiff;
 Categories=Application;Office;
