diff --git a/po/POTFILES.skip b/po/POTFILES.skip
index 26f7de7..4ad8122 100644
--- a/po/POTFILES.skip
+++ b/po/POTFILES.skip
@@ -1 +1,9 @@
 plugins/vorbis/vcedit.c
+.pc/changelog_fix.patch/plugins/gtkui/callbacks.c
+.pc/debian_native_copying.patch/plugins/gtkui/callbacks.c
+optmath.h
+plugins/adplug/adplug/realopl.cpp
+.pc/shellexec.diff/plugins/shellexecui/shellexecui.c
+plugins/supereq/nsfft-1.00/simd/SIMDBase.c
+plugins/uade2/uade-2.13/src/include/compiler.h
+tools/apbuild/ctype.h
