--- deadbeef-devel-0.5.6+git20121108.orig/po/POTFILES.in
+++ deadbeef-devel-0.5.6+git20121108/po/POTFILES.in
@@ -34,3 +34,10 @@ plugins/shellexecui/support.c
 plugins/shellexecui/shellexec.glade
 plugins/gtkui/actionhandlers.c
 plugins/gtkui/ctmapping.c
+plugins/gtkui/widgets.c
+plugins/gtkui/actionhandlers.c
+plugins/gtkui/hotkeys.c
+plugins/gtkui/pltmenu.c
+plugins/pltbrowser/pltbrowser.c
+plugins/gtkui/smclient/eggdesktopfile.c
+plugins/gtkui/smclient/eggsmclient.c
