--- chiaki-1.1.1.orig/gui/CMakeLists.txt
+++ chiaki-1.1.1/gui/CMakeLists.txt
@@ -103,7 +103,7 @@ set_target_properties(chiaki PROPERTIES
 		RESOURCE "${RESOURCE_FILES}")
 
 install(TARGETS chiaki
-		RUNTIME DESTINATION bin
-		BUNDLE DESTINATION bin)
+		RUNTIME DESTINATION games
+		BUNDLE DESTINATION games)
 install(FILES chiaki.desktop DESTINATION share/applications)
-install(FILES chiaki.png DESTINATION share/icons/hicolor/512x512)
+install(FILES chiaki.png DESTINATION share/icons/hicolor/512x512/apps)
