#! /bin/sh /usr/share/dpatch/dpatch-run

diff -Naur pytyle-0.7.5.orig/pytyle pytyle-0.7.5/pytyle
--- pytyle-0.7.5.orig/pytyle	2010-10-15 19:51:38.000000000 +0400
+++ pytyle-0.7.5/pytyle	2010-11-02 19:58:56.697098610 +0300
@@ -81,7 +81,8 @@
     if not os.access(config_file, os.F_OK | os.R_OK):
         if not os.path.exists(config_path):
             os.makedirs(config_path)
-        shutil.copyfile("%s/PyTyle/pytylerc" % (distutils.sysconfig.get_python_lib()), config_file)
+        pathpytylerc = '/usr/share/pytyle'
+        shutil.copyfile("%s/pytylerc" % (pathpytylerc), config_file)
 
     if os.access(config_file, os.F_OK | os.R_OK):
         execfile(config_file)
