Description: Fix work with libjson-rpc-perl >= 1.00
Origin: https://github.com/eiskaltdcpp/eiskaltdcpp/commit/1d41324
Applied-Upstream: 2.3.0
Last-Update: 2014-01-02


--- a/eiskaltdcpp-cli/cli-jsonrpc.pl
+++ b/eiskaltdcpp-cli/cli-jsonrpc.pl
@@ -87,15 +87,16 @@
 my $res;
 
 # creating and configuring jsonrpc client
-my $client = eval
+my $client;
+eval
 {
 	require JSON::RPC::Client;
-	JSON::RPC::Client->new();
+	$client = JSON::RPC::Client->new();
 }
 or do
 {
 	require JSON::RPC::Legacy::Client;
-	JSON::RPC::Legacy::Client->new();
+	$client=JSON::RPC::Legacy::Client->new();
 };
 $client->version("2.0");
 $client->ua->timeout(10);
