--- ocropus-0.6.0+hg20130905.orig/ocrolib/common.py
+++ ocropus-0.6.0+hg20130905/ocrolib/common.py
@@ -681,10 +681,10 @@ data_paths = [
     "./models",
     "./data",
     "./gui",
-    "/usr/local/share/ocropus/models",
-    "/usr/local/share/ocropus/data",
-    "/usr/local/share/ocropus/gui",
-    "/usr/local/share/ocropus",
+    "/usr/share/ocropus/models",
+    "/usr/share/ocropus/data",
+    "/usr/share/ocropus/gui",
+    "/usr/share/ocropus",
 ]
 
 def ocropus_find_file(fname,gz=1):
--- ocropus-0.6.0+hg20130905.orig/ocrolib/default.py
+++ ocropus-0.6.0+hg20130905/ocrolib/default.py
@@ -16,7 +16,7 @@ installable += ["en-mixed-round1.lineest
 installable += ["frakant.pyrnn.gz"] # Fraktur recognizer
 installable += ["fraktur.pyrnn.gz"] # Fraktur recognizer
 
-modeldir = "/usr/local/share/ocropus/"
+modeldir = "/usr/share/ocropus/"
 
 import os
 def getlocal():
