Subject: fix an error when QtSql doesn't support multiple statements at a time
Author: David Sansome
Origin: upstream, http://code.google.com/p/clementine-player/source/detail?r=3ade9217d73b

---
 data/schema/schema.sql |    1 +
 1 file changed, 1 insertion(+)

Index: clementine-1.0.1+dfsg/data/schema/schema.sql
===================================================================
--- clementine-1.0.1+dfsg.orig/data/schema/schema.sql
+++ clementine-1.0.1+dfsg/data/schema/schema.sql
@@ -35,4 +35,5 @@
 CREATE TABLE schema_version (
   version INTEGER NOT NULL
 );
+
 INSERT INTO schema_version (version) VALUES (0);
