Description: debian native copying
--- deadbeef-0.6.2+git20151211.orig/plugins/gtkui/callbacks.c
+++ deadbeef-0.6.2+git20151211/plugins/gtkui/callbacks.c
@@ -376,7 +376,7 @@ on_gpl1_activate                       (
                                         gpointer         user_data)
 {
     char fname[PATH_MAX];
-    snprintf (fname, sizeof (fname), "%s/%s", deadbeef->get_system_dir(DDB_SYS_DIR_DOC), "COPYING.GPLv2");
+    snprintf (fname, sizeof (fname), "/usr/share/common-licenses/%s","GPL-2");
     gtkui_show_info_window (fname, "GNU GENERAL PUBLIC LICENSE Version 2", &gplwindow);
 }
 
@@ -387,7 +387,7 @@ on_lgpl1_activate                      (
                                         gpointer         user_data)
 {
     char fname[PATH_MAX];
-    snprintf (fname, sizeof (fname), "%s/%s", deadbeef->get_system_dir(DDB_SYS_DIR_DOC), "COPYING.LGPLv2.1");
+    snprintf (fname, sizeof (fname), "/usr/share/common-licenses/%s", "LGPL-2.1");
     gtkui_show_info_window (fname, "GNU LESSER GENERAL PUBLIC LICENSE Version 2.1", &lgplwindow);
 }
 
