From e61f2705ae766dc39082e34e3594847b6634c09c Mon Sep 17 00:00:00 2001
From: Romuald Conty <neomilium@gmail.com>
Date: Tue, 6 Jan 2015 02:11:15 +0100
Subject: [PATCH 1/4] RPluginLoader: set plugins directory

---
 src/core/RPluginLoader.cpp | 1 +
 1 file changed, 1 insertion(+)

Index: stable/src/core/RPluginLoader.cpp
===================================================================
--- stable.orig/src/core/RPluginLoader.cpp	2017-03-21 15:25:04.765512560 +0100
+++ stable/src/core/RPluginLoader.cpp	2017-03-21 15:25:34.000000000 +0100
@@ -238,6 +238,7 @@
 }
 
 QString RPluginLoader::getPluginsPath() {
+    return QString("/usr/lib/qcad/plugins");
     QDir pluginsDir = QDir(".");
     if (!pluginsDir.cd("plugins")) {
         pluginsDir.cdUp();
