From 22c0fdbb8dd75d53412605c327a2bb7c43fc989f Mon Sep 17 00:00:00 2001
From: Romuald Conty <neomilium@gmail.com>
Date: Tue, 6 Jan 2015 02:12:08 +0100
Subject: [PATCH 3/4] RS: set shared data directory

---
 src/core/RS.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/RS.cpp b/src/core/RS.cpp
index d3f51da..5703b43 100644
--- a/src/core/RS.cpp
+++ b/src/core/RS.cpp
@@ -79,7 +79,7 @@ bool RS::compare(const QPair<QVariant, RPropertyAttributes>& p1, const QPair<
 QStringList RS::getDirectoryList(const QString& subDirectory) {
     QStringList dirList;
 
-    QString appDir = QCoreApplication::applicationDirPath();
+    QString appDir("/usr/share/qcad");
     QFileInfo fi(appDir);
     if (fi.fileName() == "debug" || fi.fileName() == "release") {
         appDir = fi.absolutePath();
-- 
1.9.1

