--- /dev/null
+++ qcad-3.15.5.8/src/3rdparty/qt-labs-qtscriptgenerator-5.12.5/qt-labs-qtscriptgenerator-5.12.5.pro
@@ -0,0 +1,4 @@
+include( ../../../shared.pri )
+
+SUBDIRS = ../qt-labs-qtscriptgenerator-5.5.0/qtbindings
+TEMPLATE = subdirs

--- /dev/null
+++ qcad-3.15.5.8/src/3rdparty/qt-labs-qtscriptgenerator-5.12.4/qt-labs-qtscriptgenerator-5.12.4.pro
@@ -0,0 +1,4 @@
+include( ../../../shared.pri )
+
+SUBDIRS = ../qt-labs-qtscriptgenerator-5.5.0/qtbindings
+TEMPLATE = subdirs

--- /dev/null
+++ qcad-3.15.5.8/src/3rdparty/qt-labs-qtscriptgenerator-5.15.4/qt-labs-qtscriptgenerator-5.15.4.pro
@@ -0,0 +1,4 @@
+include( ../../../shared.pri )
+
+SUBDIRS = ../qt-labs-qtscriptgenerator-5.5.0/qtbindings
+TEMPLATE = subdirs

--- /dev/null
+++ qcad-3.15.5.8/src/3rdparty/qt-labs-qtscriptgenerator-5.15.6/qt-labs-qtscriptgenerator-5.15.6.pro
@@ -0,0 +1,4 @@
+include( ../../../shared.pri )
+
+SUBDIRS = ../qt-labs-qtscriptgenerator-5.5.0/qtbindings
+TEMPLATE = subdirs

--- /dev/null
+++ qcad-3.15.5.8/src/3rdparty/qt-labs-qtscriptgenerator-5.15.7/qt-labs-qtscriptgenerator-5.15.7.pro
@@ -0,0 +1,4 @@
+include( ../../../shared.pri )
+
+SUBDIRS = ../qt-labs-qtscriptgenerator-5.5.0/qtbindings
+TEMPLATE = subdirs


