Description: Add git ChangeLog
Author: Alexander Pozdnyakov <almipo@mail.ru>
Last-Update: 2018-02-17

--- /dev/null
+++ tesseract-lang-old2-4.00~git15-9b2e3f6/ChangeLog
@@ -0,0 +1,13 @@
+2017-09-14 - migrating to tessdata_best repository (commit: 9b2e3f6)
+2017-07-31 - Added best traineddatas for 4.00 alpha (commit: 3a94ddd)
+2017-06-05 - ita: Remove ita.config from ita.traineddata (commit: 71689a2)
+2016-12-28 - Add link to information in Tesseract wiki (commit: 5d7090e)
+2016-12-27 - Add README.md (commit: 4c25d86)
+2016-12-14 - Deleted cube tessdata (commit: 1575dd7)
+2016-11-28 - Added LSTM models+lang models to 101 langs (commit: 4592b8d)
+2016-03-08 - do not load Serbian Cyrillic for Serbian latin OCR (fixes #13) (commit: 3cf1e2d)
+2015-08-03 - add license info (commit: 6c96577)
+2015-06-24 - Updated 98 traineddata files with the 3.04 training. ara, eng, hin, kor not included as they regressed. (commit: 074c372)
+2014-09-07 - rename fraktur data to google tessdata pattern (commit: 4d64457)
+2014-08-12 - Initial commit of *all* the traineddata files and cube data (commit: bf82613)
+
