diff -Naur muse-1.0.orig/share/scripts/Makefile.am muse-1.0/share/scripts/Makefile.am
--- muse-1.0.orig/share/scripts/Makefile.am	2009-12-21 18:37:20.000000000 +0300
+++ muse-1.0/share/scripts/Makefile.am	2009-12-24 23:21:11.000000000 +0300
@@ -5,7 +5,7 @@
 #musemidiscriptspkgdatadir = $(pkgdatadir)/scripts
 
 install-data-local: 
-	mkdir -p $(pkgdatadir)/scripts/
-	cp $(EXTRA_DIST) $(pkgdatadir)/scripts/
-	chmod a+x $(pkgdatadir)/scripts/*
+	mkdir -p $(DESTDIR)$(pkgdatadir)/scripts/
+	cp $(EXTRA_DIST) $(DESTDIR)$(pkgdatadir)/scripts/
+	chmod a+x $(DESTDIR)$(pkgdatadir)/scripts/*
 
diff -Naur muse-1.0.orig/share/scripts/Makefile.in muse-1.0/share/scripts/Makefile.in
--- muse-1.0.orig/share/scripts/Makefile.in	2009-12-21 18:37:44.000000000 +0300
+++ muse-1.0/share/scripts/Makefile.in	2009-12-24 23:20:20.000000000 +0300
@@ -389,9 +389,9 @@
 #musemidiscriptspkgdatadir = $(pkgdatadir)/scripts
 
 install-data-local: 
-	mkdir -p $(pkgdatadir)/scripts/
-	cp $(EXTRA_DIST) $(pkgdatadir)/scripts/
-	chmod a+x $(pkgdatadir)/scripts/*
+	mkdir -p $(DESTDIR)$(pkgdatadir)/scripts/
+	cp $(EXTRA_DIST) $(DESTDIR)$(pkgdatadir)/scripts/
+	chmod a+x $(DESTDIR)$(pkgdatadir)/scripts/*
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
diff -Naur muse-1.0.orig/utils/Makefile.am muse-1.0/utils/Makefile.am
--- muse-1.0.orig/utils/Makefile.am	2009-12-21 18:37:20.000000000 +0300
+++ muse-1.0/utils/Makefile.am	2009-12-24 23:35:44.000000000 +0300
@@ -1,6 +1,6 @@
 EXTRA_DIST = muse-find-unused-wavs
 
 install-data-local:
-	cp $(EXTRA_DIST) $(bindir)/
-	chmod a+x $(bindir)/$(EXTRA_DIST) 
+	cp $(EXTRA_DIST) $(DESTDIR)$(bindir)/
+	chmod a+x $(DESTDIR)$(bindir)/$(EXTRA_DIST) 
 
diff -Naur muse-1.0.orig/utils/Makefile.in muse-1.0/utils/Makefile.in
--- muse-1.0.orig/utils/Makefile.in	2009-12-21 18:37:45.000000000 +0300
+++ muse-1.0/utils/Makefile.in	2009-12-24 23:36:08.000000000 +0300
@@ -385,8 +385,8 @@
 
 
 install-data-local:
-	cp $(EXTRA_DIST) $(bindir)/
-	chmod a+x $(bindir)/$(EXTRA_DIST) 
+	cp $(EXTRA_DIST) $(DESTDIR)$(bindir)/
+	chmod a+x $(DESTDIR)$(bindir)/$(EXTRA_DIST) 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
