Subject: fix plugins to load icons from FHS path where they
 are installed by packaging.
Author: Ricardo Mones <mones@debian.org>
diff -urN launchy-2.5.orig/plugins/calcy/calcy.cpp launchy-2.5/plugins/calcy/calcy.cpp
--- launchy-2.5.orig/plugins/calcy/calcy.cpp	2010-04-04 04:46:15.000000000 +0200
+++ launchy-2.5/plugins/calcy/calcy.cpp	2012-05-21 13:26:41.000000000 +0200
@@ -200,7 +200,7 @@
 
 QString calcyPlugin::getIcon()
 {
-	return libPath + "/icons/calcy.png";
+	return "/usr/share/launchy/plugins/icons/calcy.png";
 }
 
 
diff -urN launchy-2.5.orig/plugins/controly/controly.cpp launchy-2.5/plugins/controly/controly.cpp
--- launchy-2.5.orig/plugins/controly/controly.cpp	2010-03-30 00:18:36.000000000 +0200
+++ launchy-2.5/plugins/controly/controly.cpp	2012-05-21 17:13:53.000000000 +0200
@@ -70,12 +70,12 @@
 
 QString controlyPlugin::getIcon()
 {
-	return getIconPath() + "/plugins/icons/controly.png";
+	return getIconPath() + "controly.png";
 }
 
 QString controlyPlugin::getIconPath() const
 {
-	return libPath + "/icons/";
+	return "/usr/share/launchy/plugins/icons/";
 }
 
 
@@ -324,4 +324,4 @@
 	return handled;
 }
 
-Q_EXPORT_PLUGIN2(controly, controlyPlugin)
\ No newline at end of file
+Q_EXPORT_PLUGIN2(controly, controlyPlugin)
diff -urN launchy-2.5.orig/plugins/gcalc/gcalc.cpp launchy-2.5/plugins/gcalc/gcalc.cpp
--- launchy-2.5.orig/plugins/gcalc/gcalc.cpp	2010-02-14 20:27:52.000000000 +0100
+++ launchy-2.5/plugins/gcalc/gcalc.cpp	2012-05-21 16:03:43.000000000 +0200
@@ -143,7 +143,7 @@
 
 QString gcalcPlugin::getIcon()
 {
-	return libPath + "/icons/calcy.png";
+	return "/usr/share/launchy/plugins/icons/calcy.png";
 }
 
 
diff -urN launchy-2.5.orig/plugins/loader/loader.cpp launchy-2.5/plugins/loader/loader.cpp
--- launchy-2.5.orig/plugins/loader/loader.cpp	2008-12-15 01:48:42.000000000 +0100
+++ launchy-2.5/plugins/loader/loader.cpp	2012-05-21 16:08:06.000000000 +0200
@@ -74,7 +74,7 @@
 
 QString loaderPlugin::getIcon()
 {
-	return libPath + "/icons/loader.png";
+	return "/usr/share/launchy/plugins/icons/loader.png";
 }
 
 
diff -urN launchy-2.5.orig/plugins/runner/runner.cpp launchy-2.5/plugins/runner/runner.cpp
--- launchy-2.5.orig/plugins/runner/runner.cpp	2010-02-15 23:03:53.000000000 +0100
+++ launchy-2.5/plugins/runner/runner.cpp	2012-05-21 16:09:59.000000000 +0200
@@ -86,7 +86,7 @@
 
 QString RunnerPlugin::getIcon()
 {
-    return libPath + "/icons/runner.png";
+    return "/usr/share/launchy/plugins/icons/runner.png";
 }
 
 
diff -urN launchy-2.5.orig/plugins/verby/Verby.cpp launchy-2.5/plugins/verby/Verby.cpp
--- launchy-2.5.orig/plugins/verby/Verby.cpp	2010-03-31 14:29:11.000000000 +0200
+++ launchy-2.5/plugins/verby/Verby.cpp	2012-05-21 16:13:10.000000000 +0200
@@ -85,7 +85,7 @@
 
 QString VerbyPlugin::getIconPath() const
 {
-	return libPath + "/icons/";
+	return "/usr/share/launchy/plugins/icons/";
 }
 
 
diff -urN launchy-2.5.orig/plugins/weby/weby.cpp launchy-2.5/plugins/weby/weby.cpp
--- launchy-2.5.orig/plugins/weby/weby.cpp	2010-03-17 03:21:04.000000000 +0100
+++ launchy-2.5/plugins/weby/weby.cpp	2012-05-21 16:13:54.000000000 +0200
@@ -392,7 +392,7 @@
 
 QString WebyPlugin::getIcon()
 {
-	return libPath + "/icons/weby.png";
+	return "/usr/share/launchy/plugins/icons/weby.png";
 }
 
 
