Description: Add git ChangeLog
Author: Alexander Pozdnyakov <almipo@mail.ru>
Last-Update: 2018-02-17

--- /dev/null	2018-05-15 20:11:39.625425796 +0300
+++ tesseract-lang-best-4.00~git23-ce12640/ChangeLog	2018-05-15 22:35:57.752771965 +0300
@@ -0,0 +1,16 @@
+2018-04-25 - correct name kur_ara to kmr - Kurmanji (Latin script) (commit: 788e2fe)
+2018-04-09 - Fix config file for Korean, remove `tessedit_load_sublangs chi_tra` (commit: c7c86bb)
+2018-03-10 - Move trained data for scripts to new subdirectory (commit: a2f7ced)
+2018-02-27 - Add Devanagari config file to fix auto PSM issue #1273 (commit: 84bd10e)
+2018-02-02 - Rename license file (commit: e744fa9)
+2018-02-02 - README: Improve description and add link to Tesseract wiki (commit: 9963c18)
+2018-02-02 - README: Add text from former COPYRIGHT and add links (commit: fb9ae6b)
+2018-02-02 - Use the full Apache License text (commit: 4928952)
+2018-02-01 - deu: Remove unwanted dependency (commit: ed5410b)
+2017-09-15 - Use legacy Orientation Script Detector (OSD) because that is the only thing that currently works. (commit: f1d1268)
+2017-09-15 - Fix Config files to LSTM only for nep and mar (commit: 9c5c2cb)
+2017-09-15 - Fix config files from Use Tesseract/LSTM combiner to LSTM only (commit: 09e4326)
+2017-09-14 - add license info (commit: c222ed8)
+2017-09-14 - Initial import (on behalf of Ray) (commit: 9ddc24e)
+2017-09-11 - Initial commit (commit: 549354e)
+

