Description: Add Debian hardening flags to qmake.
Author: Mateusz Łukasik <mati75@linuxmint.pl>

 
--- smplayer-18.6.0+svn9144.orig/src/smplayer.pro
+++ smplayer-18.6.0+svn9144/src/smplayer.pro
@@ -36,6 +36,11 @@ DEFINES += INITIAL_BLACKBORDERS
 DEFINES += CHROMECAST_SUPPORT
 DEFINES += USE_QRCODE
 
+QMAKE_CPPFLAGS *= $(shell dpkg-buildflags --get CPPFLAGS)
+QMAKE_CFLAGS   *= $(shell dpkg-buildflags --get CFLAGS)
+QMAKE_CXXFLAGS *= $(shell dpkg-buildflags --get CXXFLAGS)
+QMAKE_LFLAGS   *= $(shell dpkg-buildflags --get LDFLAGS)
+
 DEFINES += MPV_SUPPORT
 DEFINES += MPLAYER_SUPPORT
 
