Description: Disable fetching potentially insecure javascript from youtube.com
Author: Reinhard Tartler <siretart@debian.org>
Bug-Debian: http://bugs.debian.org/870233
Last-update: 2018-06-19

--- a/src/smplayer.pro
+++ b/src/smplayer.pro
@@ -436,7 +436,7 @@ contains( DEFINES, YOUTUBE_SUPPORT ) {
 	SOURCES += youtube/retrieveyoutubeurl.cpp youtube/loadpage.cpp
 
 	contains( DEFINES, YT_USE_SCRIPT ) {
-		DEFINES += YT_USE_SIG
+		#DEFINES += YT_USE_SIG # disabled 20180619, cf. http://bugs.debian.org/870233 -- rt
 		#DEFINES += YT_USE_YTSIG
 		QT += script
 	}
