--- deadbeef-0.7.0+git20160131.orig/plugins/gtkui/callbacks.c
+++ deadbeef-0.7.0+git20160131/plugins/gtkui/callbacks.c
@@ -371,7 +371,7 @@ on_changelog1_activate                 (
     char s[200];
     snprintf (s, sizeof (s), _("DeaDBeeF %s ChangeLog"), VERSION);
     char fname[PATH_MAX];
-    snprintf (fname, sizeof (fname), "%s/%s", deadbeef->get_doc_dir (), "ChangeLog");
+    snprintf (fname, sizeof (fname), "%s/%s", deadbeef->get_system_dir(DDB_SYS_DIR_DOC), "changelog.deadbeef");
     gtkui_show_info_window (fname, s, &changelogwindow);
 }
