--- a/plugins/gtkui/callbacks.c
+++ b/plugins/gtkui/callbacks.c
@@ -745,7 +745,7 @@
     char s[200];
     snprintf (s, sizeof (s), _("DeaDBeeF %s ChangeLog"), VERSION);
     char fname[PATH_MAX];
-    snprintf (fname, sizeof (fname), "%s/%s", deadbeef->get_doc_dir (), "ChangeLog");
+    snprintf (fname, sizeof (fname), "%s/%s", deadbeef->get_doc_dir (), "changelog.deadbeef");
     show_info_window (fname, s, &changelogwindow);
 }
 
