--- deadbeef-0.7.2+git7396-1f8f7f3e.orig/plugins/gtkui/Makefile.am
+++ deadbeef-0.7.2+git7396-1f8f7f3e/plugins/gtkui/Makefile.am
@@ -90,13 +90,13 @@ ddb_gui_GTK3_la_LIBADD = $(LDADD) $(GTK3
 ddb_gui_GTK3_la_CFLAGS = -std=c99 $(GTK3_DEPS_CFLAGS) $(SM_CFLAGS) $(JANSSON_CFLAGS)
 ddb_gui_GTK3_la_OBJCFLAGS = $(GTK3_DEPS_CFLAGS) $(SM_CFLAGS) $(JANSSON_CFLAGS)
 
-nodist_ddb_gui_GTK3_la_SOURCES = temp/gtkui-gresources.c
+nodist_ddb_gui_GTK3_la_SOURCES = gtkui-gresources.c
 
 resource_files = \
 	gtkui.gresources.xml	\
 	gtk/menus.ui
 
-temp/gtkui-gresources.c: $(resource_files)
+gtkui-gresources.c: $(resource_files)
 	$(AM_V_GEN) $(GLIB_COMPILE_RESOURCES) --target=$@ --sourcedir=$(srcdir) --generate-source $<
 
 endif
