diff -Naur deadbeef-0.5.0+git20110516/po/POTFILES.in deadbeef-0.5.0+git201105161/po/POTFILES.in
--- deadbeef-0.5.0+git20110516/po/POTFILES.in	2011-05-17 19:00:34.000000000 +0400
+++ deadbeef-0.5.0+git201105161/po/POTFILES.in	2011-05-17 19:37:39.086327820 +0400
@@ -25,4 +25,8 @@
 plugins/converter/support.c
 plugins/gtkui/dspconfig.c
 plugins/gtkui/fileman.c
-
+plugins/converter/converter.glade
+plugins/supereq/nsfft-1.00/simd/SIMDBase.c
+plugins/uade2/uade-2.13/src/include/compiler.h
+tools/apbuild/apsymbols.h
+tools/apbuild/ctype.h
