diff -Naur deadbeef-0.5.0+git20110516/po/POTFILES.in deadbeef-0.5.0+git201105161/po/POTFILES.in
--- deadbeef-0.5.2+git20120419.orig/po/POTFILES.in
+++ deadbeef-0.5.2+git20120419/po/POTFILES.in
@@ -35,3 +35,7 @@
 plugins/gtkui/pltmenu.c
 plugins/pltbrowser/pltbrowser.c
 tf.c
+plugins/rg_scanner/ddb_rg_scan_gui.c
+plugins/rg_scanner/ddb_rg_scan_gui.glade
+plugins/rg_scanner/interface.c
+plugins/rg_scanner/support.c
