--- deadbeef-0.5.2+git20120419.orig/plugins/shellexecui/shellexecui.c
+++ deadbeef-0.5.2+git20120419/plugins/shellexecui/shellexecui.c
@@ -17,9 +17,9 @@
 */
 #include <stdlib.h>
 #include <string.h>
-#include <deadbeef/deadbeef.h>
+#include "../../deadbeef.h"
 #include <gtk/gtk.h>
-#include <deadbeef/gtkui_api.h>
+#include "../gtkui/gtkui_api.h"
 #include <glib/gprintf.h>
 #include "interface.h"
 #include "callbacks.h"
