Description: Add hardening flags into compiler options
Author: Boris Pek <tehnick-8@mail.ru>
Last-Update: 2014-07-25

--- a/src/src.pro
+++ b/src/src.pro
@@ -156,6 +156,12 @@
 RESOURCES += ../psi.qrc ../iconsets.qrc
 
 # Platform specifics
+unix:!mac {
+	QMAKE_CXXFLAGS += $(Q_CXXFLAGS)
+	QMAKE_CFLAGS_DEBUG += $(Q_CFLAGS)
+	QMAKE_CFLAGS_RELEASE += $(Q_CFLAGS)
+	QMAKE_LFLAGS += $(Q_LDFLAGS)
+}
 windows {
     RC_ICONS = ../win32/app.ico
 }
