--- tesseract-4.00~git2107-6f13d755.orig/doc/Makefile.am
+++ tesseract-4.00~git2107-6f13d755/doc/Makefile.am
@@ -8,8 +8,12 @@ man_MANS = cntraining.1  combine_tessdat
 
 EXTRA_DIST = $(man_MANS) Doxyfile
 
+man_xslt = /usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/docbook.xsl
+
 %: %.asc
-	$(asciidoc) -o $@ $<
+	$(asciidoc) $<
+	$(asciidoc) -b docbook $<
+	xsltproc $(man_xslt) $@.xml
 
 MAINTAINERCLEANFILES = $(man_MANS) Doxyfile
 
