Description: Fixing spelling in man page
Author: Alexander Pozdnyakov <almipo@mail.ru>
Last-Update: 2018-02-17

--- tesseract-4.00~git2198-ce7ee87f.orig/doc/combine_tessdata.1.asc
+++ tesseract-4.00~git2198-ce7ee87f/doc/combine_tessdata.1.asc
@@ -61,7 +61,7 @@ OPTIONS
 
 *-o* '.traineddata' 'FILE'...:
     Overwrites the specified components of the .traineddata file
-    with those provided on the comand line.
+    with those provided on the command line.
 
 *-u* '.traineddata' 'PATHPREFIX'
     Unpacks the .traineddata using the provided prefix.
