Author: Philip Rinn <rinni@inventati.org>
Description:  Fix build with CMake >= 3.17
Forwarded: not-needed
Last-update: 2020-09-08
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,7 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.11)
+IF(${CMAKE_VERSION} VERSION_GREATER "3.17.0")
+    CMAKE_POLICY(SET CMP0100 NEW)
+ENDIF()
 LIST(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 INCLUDE(GSettings)
 SET(ENV{LC_ALL} "C")
