--- azpainter-2.1.5.orig/AUTHORS
+++ azpainter-2.1.5/AUTHORS
@@ -2,11 +2,11 @@
 -------------------
 
 be
-    Zmicer Turok <zmicerturok@gmail.com>, 2018
+    Zmicer Turok <zmicerturok@gmail.com>, 2018-2019
 
 en
-    Alexey Dokuchaev <danfe@nsu.ru>, 2017-2018
-    Eugeniy Fedirets <evgeniy_fedirets@ukr.net>, 2017-2018
+    Alexey Dokuchaev <danfe@nsu.ru>, 2017-2019
+    Eugeniy Fedirets <evgeniy_fedirets@ukr.net>, 2017-2019
 
 es_ES
     David
@@ -15,7 +15,7 @@
     Konrad Stahlhofer, 2020
 
 fr
-    lbs <uselessmailplease@free.fr>, 2018
+    lbs <uselessmailplease@free.fr>, 2018-2019
 
 it
     Nihiles <nihiles@keemail.me>, 2019-2020
@@ -24,11 +24,12 @@
     Adriano Duarte
 
 ru
-    Eugeniy Fedirets <evgeniy_fedirets@ukr.net>, 2017
-    Alexandre Prokoudine <alexandre.prokoudine@gmail.com>, 2017
+    Alexey Dokuchaev <danfe@nsu.ru>, 2019
+    Eugeniy Fedirets <evgeniy_fedirets@ukr.net>, 2017-2019
+    Alexandre Prokoudine <alexandre.prokoudine@gmail.com>, 2017-2019
 
 uk
-    Eugeniy Fedirets <evgeniy_fedirets@ukr.net>, 2018
+    Eugeniy Fedirets <evgeniy_fedirets@ukr.net>, 2018-2019
 
 zh_CN
     SenlinOS (森林OS) <comics2008@gmail.com>, 2017-2019
