--- azpainter-2.0.2.orig/desktop/azpainter.desktop
+++ azpainter-2.0.2/desktop/azpainter.desktop
@@ -1,8 +1,14 @@
 [Desktop Entry]
-Version=1.0
+Version=2.0
 Type=Application
 Name=AzPainter
-Exec=azpainter
+Exec=azpainter %F
 Icon=azpainter
 Terminal=false
 Categories=Graphics;
+Comment=16bit RGB full color paint software.
+Comment[ja]=16bit RGB full color paint software.
+Comment[pt_BR]=16bit RGB full color paint software.
+Comment[ru]=16bit RGB full color paint software.
+Comment[uk]=16bit RGB full color paint software.
+MimeType=application/x-azpainter-apd;
