Only in compizboxmenu: .git
diff -up compizboxmenu-1.1.1+git20120207/Makefile compizboxmenu/Makefile
--- compizboxmenu-1.1.1+git20120207/Makefile	2012-03-09 22:44:57.000000000 -0700
+++ compizboxmenu/Makefile	2012-07-31 11:33:48.280557022 -0600
@@ -10,20 +10,20 @@ LDFLAGS_CLIENT := `pkg-config --libs dbu
 all: compiz-boxmenu-daemon compiz-boxmenu compiz-boxmenu-dlist compiz-boxmenu-vplist compiz-boxmenu-wlist
 
 compiz-boxmenu: deskmenu.c deskmenu-common.h
-	$(CC) $(CPPFLAGS_CLIENT) $(CFLAGS) $(LDFLAGS_CLIENT) -o $@ $<
+	$(CC) $(CPPFLAGS_CLIENT) $(CFLAGS) -o $@ $< $(LDFLAGS_CLIENT)  
 
 compiz-boxmenu-dlist: deskmenu-documentlist-client.c deskmenu-common.h
-	$(CC) $(CPPFLAGS_CLIENT) $(CFLAGS) $(LDFLAGS_CLIENT) -o $@ $<
+	$(CC) $(CPPFLAGS_CLIENT) $(CFLAGS) -o $@ $< $(LDFLAGS_CLIENT)  
 
 compiz-boxmenu-vplist: deskmenu-vplist-client.c deskmenu-common.h
-	$(CC) $(CPPFLAGS_CLIENT) $(CFLAGS) $(LDFLAGS_CLIENT) -o $@ $<
+	$(CC) $(CPPFLAGS_CLIENT) $(CFLAGS) -o $@ $< $(LDFLAGS_CLIENT)  
 
 compiz-boxmenu-wlist: deskmenu-windowlist-client.c deskmenu-common.h
-	$(CC) $(CPPFLAGS_CLIENT) $(CFLAGS) $(LDFLAGS_CLIENT) -o $@ $<
+	$(CC) $(CPPFLAGS_CLIENT) $(CFLAGS) -o $@ $< $(LDFLAGS_CLIENT)  
 
 compiz-boxmenu-daemon: deskmenu-menu.c deskmenu-wnck.c deskmenu-utils.c deskmenu-utils.h deskmenu-wnck.h deskmenu-glue.h deskmenu-common.h deskmenu-menu.h
 
-	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -o $@ deskmenu-menu.c deskmenu-wnck.c deskmenu-utils.c
+	$(CC) $(CPPFLAGS) $(CFLAGS) -o $@ deskmenu-menu.c deskmenu-wnck.c deskmenu-utils.c  $(LDFLAGS)  
 
 deskmenu-glue.h: deskmenu-service.xml
 	dbus-binding-tool --mode=glib-server --prefix=deskmenu --output=$@ $^
Only in compizboxmenu: deskmenu-common.h.gch
Only in compizboxmenu-1.1.1+git20120207: deskmenu-glue.h
diff -up compizboxmenu-1.1.1+git20120207/deskmenu-menu.c compizboxmenu/deskmenu-menu.c
--- compizboxmenu-1.1.1+git20120207/deskmenu-menu.c	2012-03-09 22:44:57.000000000 -0700
+++ compizboxmenu/deskmenu-menu.c	2012-07-31 12:12:29.388579378 -0600
@@ -163,7 +163,7 @@ pipe_menu_recreate (GtkWidget *item, gch
 			g_markup_parse_context_free (context);
 			if (error)
 			{
-				g_print(error->message); //spit out the message manually
+				g_print("%s", error->message); //spit out the message manually
 				if (dm_object->current_item)
 				{
 					//force reset
@@ -1300,10 +1300,12 @@ deskmenu_reload (Deskmenu *deskmenu,
 gboolean
 deskmenu_control (Deskmenu *deskmenu, gchar *filename, gchar *workingd, GError  **error)
 {
-	chdir (workingd);
-	DeskmenuObject *dm_object = check_file_cache (deskmenu, g_strstrip(filename));
-	deskmenu_show(dm_object, deskmenu, error);
-	return TRUE;
+	if(chdir (workingd) == 0) {
+		DeskmenuObject *dm_object = check_file_cache (deskmenu, g_strstrip(filename));
+		deskmenu_show(dm_object, deskmenu, error);
+		return TRUE;
+	}
+	return FALSE;
 }
 
 //precache backend, currently needs GUI
@@ -1315,7 +1317,7 @@ deskmenu_precache (Deskmenu *deskmenu, g
 	guint i = 0;
 	gsize total = 0;
 
-	(void *)check_file_cache(deskmenu, ""); //always cache default menu
+	check_file_cache(deskmenu, ""); //always cache default menu
 
 	g_print("Attempting to precache files in config...");
 	if (!filename)
@@ -1335,17 +1337,16 @@ deskmenu_precache (Deskmenu *deskmenu, g
 	{
 		g_print("Configuration found! Starting precache...");
 		gchar **files = g_key_file_get_keys (config, "Files", &total, &error);
-		gchar *feed = "";
-		for (;i < total;i++)
+		gchar *feed = NULL;
+		for (i = 0;i < total;i++)
 		{
 			feed = g_key_file_get_string (config, "Files", files[i], &error);
-			(void *)check_file_cache(deskmenu, parse_expand_tilde(feed));
+			if (feed) {
+				check_file_cache(deskmenu, parse_expand_tilde(feed));
+				g_free(feed);
+			}
 		}
 		g_strfreev(files);
-		if (feed != "")
-		{
-			g_free(feed);
-		}
 	}
 	g_key_file_free (config);
 }
Common subdirectories: compizboxmenu-1.1.1+git20120207/hicolor and compizboxmenu/hicolor
Common subdirectories: compizboxmenu-1.1.1+git20120207/new-editor and compizboxmenu/new-editor
